src/libcore/convert.rs

```diff
@@ -17,6 +17,26 @@
 //! Like many traits, these are often used as bounds for generic functions, to
 //! support arguments of multiple types.
 //!
+//! - Use `as` for reference-to-reference conversions
```

Comment:
The RFC which introduced them doesn't agree ([ref](https://github.com/rust-lang/rfcs/blob/master/text/0529-conversion-traits.md#basic-design)) and has a nice rationale about it. If the rfc is still right, maybe explain further the motivation ?